tests/python/LookTest.py

```diff
@@ -84,6 +87,7 @@ def test_transform(self):
         # Wrong type tests.
         with self.assertRaises(TypeError):
             self.look.setTransform(OCIO.ALLOCATION_UNIFORM)
+            self.look.setTransform(1)
```

Comment:
These added second lines under `assertRaises` won't ever be executed, since the previous line will raise an exception. Applies to all uses in this module. Perhaps instead do:

```
for invalid in (..., ...):
    with self.assertRaises(TypeError):
        self.look.setSomething(invalid)
```

Reply:
Resolved in b8a3feb.